packages/web-app-files/src/mixins/spaces/actions/uploadImage.js

```diff
@@ -38,10 +39,21 @@ export default {
       this.$data.$_uploadImage_selectedSpace = resources[0]
       this.$refs.spaceImageInput.click()
     },
-    $_uploadImage_uploadImageSpace(ev) {
+    $_uploadImage_uploadImageSpace: function (ev) {
       const graphClient = clientService.graphAuthenticated(this.configuration.server, this.getToken)
       const file = ev.currentTarget.files[0]
 
+      if (
+        !ThumbnailService.SupportedMimeTypes.filter((mimeType) =>
+          mimeType.startsWith('image/')
+        ).includes(file.type)
+      ) {
+        return this.showMessage({
+          title: this.$gettext('Failed to upload space image'),
```

Comment:
Please give the user context (`desc` key) why the upload failed. In this case you know that the file type is unsupported. A description like `The file type is unsupported` would be beneficial.